src/main/java/no/nav/fo/veilarbregistrering/registrering/bruker/BrukerRegistreringService.java

```diff
@@ -214,6 +215,19 @@ private OrdinaerBrukerRegistrering mottattBruker(Bruker bruker, OrdinaerBrukerRe
         return ordinaerBrukerRegistrering;
     }
 
+    public void oppdaterRegistreringTilstand(RegistreringTilstandDto registreringTilstandDto) {
+        RegistreringTilstand original = brukerRegistreringRepository.hentRegistreringTilstand(registreringTilstandDto.getId());
+        RegistreringTilstand oppdatert = original.oppdaterStatus(registreringTilstandDto.getStatus());
+        brukerRegistreringRepository.oppdater(RegistreringTilstand.of(
```

Comment:
Hvorfor sender du ikke bare inn den oppdaterte, i stedet for å lage en ny en?

Reply:
Ønsker å tenke mer funksjonelt når jeg koder og tenke på objekter mer som immutable 😊

Reply:
Det var det allerede - oppdaterStatus laget et nytt objekt som du fikk tilbake :)